src/Microsoft.Extensions.StackTrace.Sources/PortablePdbReader.cs

```diff
@@ -107,6 +107,17 @@ private static string GetPdbPath(string assemblyPath)
             return null;
         }
 
+        private static bool IsPortable(string pdbPath)
+        {
+            using (var pdbStream = File.OpenRead(pdbPath))
```

Comment:
It was to avoid trying to conditionally seek the stream (in the event the stream wasn't seekable).